.github/workflows/build.yml

```diff
@@ -26,6 +28,11 @@ jobs:
       with:
         go-version: ${{ matrix.go-version }}
 
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
+      with:
+        cosign-release: 'v1.4.1'
+
```

Comment:
here is just the build we are not running signing and anything that push things just to test.
the release job will come next and then we can sign :)